### PR TITLE
Update routing.rst

### PR DESCRIPTION
**Pull request by DurandSacha**

### CODE REVIEW

**Review by DurandSacha — COMMENTED** [review]
I just add a json return for function

File: routing.rst [hunk 1]
```diff
@@ -152,7 +152,7 @@ Use the ``methods`` option to restrict the verbs each route should respond to:
         // src/Controller/BlogApiController.php
         namespace App\Controller;
 
-        // ...
+        use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
```
Comment (HeahDude) [inline routing.rst]: Can be reverted for the same reason given in #13145

File: routing.rst [hunk 2]
```diff
@@ -161,7 +161,8 @@ Use the ``methods`` option to restrict the verbs each route should respond to:
              */
             public function show(int $id)
             {
-                // ... return a JSON response with the post
+                // create your json response
+                return $this->json('Hello Symfony');
```
Comment (HeahDude) [inline routing.rst]: This is not consistent with the route definition, I would keep it as is instead.

**Review by HeahDude — COMMENTED** [review]
Thank you for your proposal, but I'm 👎 in the current state for the given reasons.

**Pull request closed without merging**